src/utils/get-hook-script.js

```diff
@@ -109,6 +109,9 @@ module.exports = function getHookScript(hookName, relativePath, npmScriptName) {
       echo "husky > npm run -s ${npmScriptName} (node \`node -v\`)"
       echo
 
+      # Allows us to read user input below, assigns stdin to keyboard
+      exec < /dev/tty
```

Comment:
Maybe an environment variable can be a temporarily workaround for the [gui clients issue](https://github.com/typicode/husky/pull/166#issuecomment-326104859)?

something like 
```sh
if [ "$HUSKY_ENABLE_STDIN" = true ]; then
  exec < /dev/tty
fi
```

This way it's a very clear opt in, and adding something like this to `package.json` should work:
```json
{
  "scripts": {
    "precommit": "cross-env HUSKY_ENABLE_STDIN=true node /some/script"
  }
}
```

I think there isn't really any other way around this, and this is a very valid use-case for teams that exclusively use the command line. 
Further move, `HUSKY_ENABLE_STDIN` should be available under `process.env`, so easy for hook devlopers to provide non-interactive fallback.